### PR TITLE
[CHORE] updated reconciliation readme

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
[CHORE](https://bcdevex.atlassian.net/browse/CHORE)

Objective: 

Update reconciliation readme

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,4 +32,5 @@ testdata
 sqlfile.sql
 sqlfile.dbml
 
-migration-results
\ No newline at end of file
+migration-results
+streamlit
```
Comment (chelsea-EYDS) [inline .gitignore:36]: I have a folder on my local. Used for testing.

**Review by vyasworks — COMMENTED** [review]
🌟

**Review by vyasworks — APPROVED** [review]

File: docs/reconciliation.readme.md [hunk 2]
```diff
@@ -1,27 +1,244 @@
-**Overview:**
+# Reconciliation
 
-Reconciliation occurs daily for each SBC location. On each run it will check all dates for all locations starting from the “fiscal_start_period”, currently 01-01-2023, up to the current date. It will only attempt to match “pending” and “in progress” payments and deposits. If there are no such payments, the system will skip that day.
+---
 
-**POS Criteria:**
+## Introduction
+
+The goal is to simplify and automate the process that compares two set of records (TDI files to Sales) to verify that financial figures are correct and in agreement. Reconciliation is used to prevent balance sheet errors on financial accounts, check for fraud, and to reconcile the general ledger.
```
Comment (vyasworks) [inline docs/reconciliation.readme.md:7]: to transactions?

Comment (vyasworks) [reply docs/reconciliation.readme.md:7]: provide link to an external definition on the web?

**Comment by vyasworks** [comment]
Skimmed through, dint find anything out of the ordinary. So good to me.

**Pull request merged**